### PR TITLE
UI: Clean up Settings widget structure and styling

### PR DESCRIPTION
**Pull request by Warchamp7**
### Description
Many of the tabs in Settings right now have margins or sizings that do not match, or have widget heirarchies that vary between them. This PR makes adjustments to many containers and widgets in Settings to make their margins and appearance more consistent.

### Motivation and Context
Want settings to look better and cleaner.

obs64_2m99ot6HaD
obs64_wZwgFFfo9a

obs64_uCkJL8gLDs
obs64_ehPbopyScX

obs64_Gf89PbiYwI
obs64_0dmM4u68qt

obs64_3agSmlNVrO
obs64_GvXvXe9HAx

obs64_xdYiS6qTow
obs64_eppZFD163Y

obs64_5KlKbcoeHP
obs64_LEdgfnUpQb

obs64_BYSkTinwQj
obs64_Sp50el0sfv

### How Has This Been Tested?
Checked through all tabs of Settings, including nested stacked pages.

### Types of changes

- Tweak (non-breaking change to improve existing functionality)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: UI/forms/OBSBasicSettings.ui [hunk 1]
```diff
@@ -8418,8 +8429,6 @@
   <tabstop>lowLatencyBuffering</tabstop>
   <tabstop>baseResolution</tabstop>
   <tabstop>outputResolution</tabstop>
-  <tabstop>downscaleFilter</tabstop>
-  <tabstop>fpsType</tabstop>
```
Comment (RytoEX) [inline UI/forms/OBSBasicSettings.ui:8422]: Is this intentional? Do we need to re-add these tabstops somewhere?

**Review by RytoEX — CHANGES_REQUESTED** [review]
A question about tabstops.

File: UI/forms/OBSBasicSettings.ui (hunk 1 above)
Comment (Warchamp7) [reply UI/forms/OBSBasicSettings.ui:8422]: I definitely discarded these removals more than once and I appear to have accidentally still included them

**Comment by derrod** [comment]
The blank space between the "Auto" checkbox and spinbox looks a bit weird.

**Review by Lain-B — APPROVED** [review]

**Comment by Warchamp7** [comment]
> The blank space between the "Auto" checkbox and spinbox looks a bit weird.

Removed that and also fixed the Enhanced Broadcast label padding

**Review by RytoEX — APPROVED** [review]
Approving since this is cleaning up changes that were made for 30.2.

**Pull request merged**